app/views/home/_chat.html.erb

```diff
@@ -19,11 +19,7 @@
 <div class="d-none" data-role="message-template">
   <div class="chat-item d-flex flex-row">
     <div class="avatar-container d-flex align-items-end">
-      <a href="#">
-        <span class="w-48 avatar">
-          <i class="fa fa-user"></i>
-        </span>
-      </a>
+      <%= render 'users/avatar', user: current_user, size: 48, classes: %i[rounded-circle] %>
```

Comment:
Is rounded-circle necessary here? Shouldn't all occurrences of avatar be circles?

Reply:
If that is the case, it can be moved to the avatar partial